test/src/com/xilinx/rapidwright/design/TestNetTools.java

```diff
@@ -0,0 +1,115 @@
+            "hwct.dcp",
+            "bug701.dcp",
+            "hwct_pr1.dcp",
+            "inout.dcp",
+            "optical-flow.dcp",
+            "design_with_backslash_2021.2.dcp",
+            "design_with_backslash_2022.1.dcp",
+            "design_with_backslash_2022.2.dcp",
+            "gnl_2_4_3_1.3_gnl_3000_07_3_80_80_placed.dcp",
+            // "picoblaze4_ooc_X6Y60_X6Y65_X10Y60_X10Y65.dcp clk",          // The source cell bufgce_inst is not placed yet.
+            "picoblaze_ooc_X10Y235.dcp",
+            "picoblaze_ooc_X10Y235_2022_1.dcp",
+            // "picoblaze_ooc_X10Y235_unreadable_edif.dcp",                 // This case would call vivado to generate an readable EDIF file.
+            "picoblaze_partial.dcp clk",
+            "reduce_or_routed_7overlaps.dcp",
+            "testCopyImplementation.dcp",
+            
+            // UltraScale
+            "microblazeAndILA_3pblocks.dcp base_mb_i/clk_wiz_1/inst/clk_out1 base_mb_i/clk_wiz_1/inst/clkfbout_buf_base_mb_clk_wiz_1_0 base_mb_i/mdm_1/U0/No_Dbg_Reg_Access.BUFG_DRCK/Dbg_Clk_31 dbg_hub/inst/BSCANID.u_xsdbm_id/itck_i",
+            "microblazeAndILA_3pblocks_2024.1.dcp base_mb_i/clk_wiz_1/inst/clk_out1 base_mb_i/clk_wiz_1/inst/clkfbout_buf_base_mb_clk_wiz_1_0 base_mb_i/mdm_1/U0/No_Dbg_Reg_Access.BUFG_DRCK/Dbg_Clk_31 dbg_hub/inst/BSCANID.u_xsdbm_id/itck_i",
+
+            // Versal
+            "noc_tutorial_routed.dcp",
+            "picoblaze_2022.2.dcp",
+            "versal_cout_hq2.dcp",
+            // "two_clk_check_NetTools.dcp clk1_IBUF_BUFG clk2_IBUF_BUFG rst1 rst2",
+
+            // Series-7
+            // "bug226.dcp murax.apb3Router_1.io_mainClk",
+            // "bug349.dcp CLK_BUFG_BOT_R_X60Y48_BUFGCTRL_X0Y0_O",
+            // "bug635.dcp",
+            // "bug709.dcp clk_IBUF_BUFG",
+            // "ramb18.dcp",
+            // "routethru_luts.dcp",
+            // "routethru_pip.dcp",
+    })
+    public void testIsGlobalClockNet(String pathAndlobalClockNames) {
+        String[] tmpList = pathAndlobalClockNames.split(" ");
+        String path = tmpList[0];
+        HashSet<String> globalClockNamesFromVivado = new HashSet<>();
+        for (int i = 1; i < tmpList.length; i++) {
+            globalClockNamesFromVivado.add(tmpList[i]);
+        }
+        HashSet<String> globalClockNamesFromRWRoute = new HashSet<>();
+        Design design = RapidWrightDCP.loadDCP(path);
+        System.out.println("Series:" + design.getDevice().getSeries());
+
+        for (Net net: design.getNets()) {
+            if (NetTools.isGlobalClock(net)) {
+                globalClockNamesFromRWRoute.add(net.getName());
+            }
+        }
+        System.out.println("-----------Vivado------------");
+        for (String nameInVivado: globalClockNamesFromVivado) {
+            System.out.println(nameInVivado);
+        }
+        System.out.println("-----------RWRoute-----------");
+        for (String nameInRWRoute: globalClockNamesFromRWRoute) {
```

Comment:
```suggestion
        for (String nameInRWRoute: globalClockNamesFromNetTools) {
```
(will need to update other mentions too)

Reply:
Updated these mentions and Series-7 related mentions in https://github.com/Xilinx/RapidWright/pull/1057/commits/98917411b2064edc6b649d4bd1bc68982acc3406.